src/service/dialogflow/conv.ts

```diff
@@ -258,6 +261,10 @@ export class DialogflowConversation<
 
   /**
    * Triggers an intent of your choosing by sending a followup event from the webhook.
+   * Final response can theoretically include responses and Action SDK specific data but
+   * these will not be handled by Dialogflow nor Google Assistant respectively.
```

Comment:
More specifically, the response doesn't actually go back to Google Assistant at all and only goes back to Dialogflow. Maybe you can change it by removing the mention of Google Assistant or mention that Dialogflow will not pass it back to the Google Assistant at all.

Also would be good to mention that specifically anything set in `conv.user.storage` won't be saved since that is most commonly what people might use.